dlls/gauss.cpp

```diff
@@ -209,7 +209,8 @@ void CGauss::SecondaryAttack()
 		SendWeaponAnim( GAUSS_SPINUP );
 		m_fInAttack = 1;
 		m_flTimeWeaponIdle = UTIL_WeaponTimeBase() + 0.5;
-		m_pPlayer->m_flStartCharge = gpGlobals->time;
+		//m_pPlayer->m_flStartCharge = gpGlobals->time;
```

Comment:
I think you can just drop the lines, no need to leave them commented out.

Reply:
> draft